pyaedt/hfss.py

```diff
@@ -1987,42 +1861,43 @@ def create_voltage_source_from_objects(self, startobj, endobject, axisdir=0, sou
         ...                                "BoxVolt1", "copper")
         >>> box2 = hfss.modeler.create_box([30, 0, 10], [40, 10, 5],
         ...                                "BoxVolt2", "copper")
-        >>> v1 = hfss.create_voltage_source_from_objects("BoxVolt1", "BoxVolt2",
-        ...                                         hfss.AxisDir.XNeg,
-        ...                                         "VoltageSource")
+        >>> v1 = hfss.create_voltage_source_from_objects("BoxVolt1","BoxVolt2",hfss.AxisDir.XNeg,"VoltageSource")
         PyAEDT INFO: Connection Correctly created
-
         """
 
-        if not self.modeler.does_object_exists(startobj) or not self.modeler.does_object_exists(endobject):
+        if not self.modeler.does_object_exists(assignment) or not self.modeler.does_object_exists(reference):
             self.logger.error("One or both objects doesn't exists. Check and retry")
             return False
         if self.solution_type in ["Modal", "Terminal", "Transient Network"]:
             sheet_name, point0, point1 = self.modeler._create_sheet_from_object_closest_edge(
-                startobj, endobject, axisdir, source_on_plane
+                assignment, reference, start_direction, source_on_plane
             )
-            sourcename = self._get_unique_source_name(sourcename, "Voltage")
-            return self.create_source_excitation(sheet_name, point0, point1, sourcename, sourcetype="Voltage")
+            name = self._get_unique_source_name(name, "Voltage")
+            return self.create_source_excitation(sheet_name, point0, point1, name, source_type="Voltage")
         return False  # pragma: no cover
 
-    @pyaedt_function_handler()
-    def create_current_source_from_objects(self, startobj, endobject, axisdir=0, sourcename=None, source_on_plane=True):
+    @pyaedt_function_handler(startobj="assignment", endobject="reference", sourcename="name", axisdir="start_direction")
+    def create_current_source_from_objects(
+        self, assignment, reference, start_direction=0, name=None, source_on_plane=True
+    ):
         """Create a current source taking the closest edges of two objects.
 
         Parameters
         ----------
-        startobj :
-            Starting object for the integration line.
-        endobject :
-            Ending object for the integration line.
-        axisdir : int or :class:`pyaedt.application.Analysis.Analysis.AxisDir`, optional
-            Position of the port. It should be one of the values for ``Application.AxisDir``,
-            which are: ``XNeg``, ``YNeg``, ``ZNeg``, ``XPos``, ``YPos``, and ``ZPos``.
-            The default is ``Application.AxisDir.XNeg``.
-        sourcename : str, optional
+        assignment : str or int or :class:`pyaedt.modeler.cad.object3d.Object3d`
+            First solid connected to the current source.
```

Comment:
```suggestion
            First object connected to the current source.
```
Same here. If you decide to put "solid" in these descriptions, use "solid object" as solid is an adjective and not a noun.